index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Personal Portfolio</title>
+
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <section id="navbar">
+      <ul>
+        <li><a href="#welcome-section">Home</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#contact">Contact Me</a></li>
+        <!-- <li><a href="#welcome-section">Home</a></li> -->
+      </ul>
+    </section>
+    <section id="welcome-section">
+      <div class="container welcome-text">
+        <h1>Hello, you are welcome!</h1>
+      </div>
+    </section>
+    <section id="projects">
+      <div class="project-title">
+        <img src="" alt="" />
+        <a href="#">Click here to view project</a>
+      </div>
+      <div class="project-title">
+        <img src="" alt="" />
+        <a href="#">Click here to view project</a>
+      </div>
+      <div class="project-title">
+        <img src="" alt="" />
+        <a href="#">Click here to view project</a>
+      </div>
+      <a
+        id="profile-link"
+        href="https://github.com/normainobary2021/TrialOne"
+        target="_blank"
+        >Show All Projects</a
+      >
+    </section>
+    <section id="contact">
+      <p>Fill this form to contact me</p>
+      <form>
+        <input type="text" placeholder="Full Name" />
+        <input type="email" placeholder="Email Address" />
+        <input type="text" placeholder="Subject" />
+        <textarea
```

Comment:
It would be much better had these form items wrapped within unordered list. It would make them better organized.

Reply:
- [ ] Good job so far for using the inline comments :+1:  But please phrase the comments politely, as suggestions instead of orders for example instead of "It would be ..." you could have said, "I think  ..."  :smile: